config/locales/en.yml

```diff
@@ -1018,6 +1018,7 @@ en:
       subhead: "Business unit: %{team_name}"
       team_your_working_with: This is the team you're working with
       will_be_joined: "%{team_name}'s users and open cases will be joined to %{target_team_name}'s business unit"
+      cannot_join_other_team: "%{team_name} cannot be joined with any other business unit"
```

Comment:
Are we happy to not keep this alphabetical, personally not an issue

Reply:
Oh good spot - there might be a few of those. If you can mark with comments like this I'll tidy them up